frontend/commondatabuilder/commondatabuilder.ts

```diff
@@ -127,15 +131,23 @@ export function createDjangoChoicesTypescriptFiles(
   });
 }
 
+function createInternationalizedStringLiteral(value: string): string {
+  const backtickedValue = "`" + JSON.stringify(value).slice(1, -1) + "`";
```

Comment:
Uhh hopefully this works?  We're only ever doing this with trusted data so it shouldn't be a security hazard; if it ever breaks we'll know ASAP and can fix it quickly.